picard/ui/options/general.py

```diff
@@ -41,6 +41,7 @@ class GeneralOptionsPage(OptionsPage):
         config.TextOption("persist", "oauth_refresh_token", ""),
         config.BoolOption("setting", "analyze_new_files", False),
         config.BoolOption("setting", "ignore_file_mbids", False),
+        config.BoolOption("setting", "cluster_new_files", False),
```

Comment:
Clustering files after each addition works slightly different than if all the files were loaded and then the cluster button was clicked ( because of the way the cluster function works if a single file is passed to it). It gives slightly more preference to "Same origin folder" based clustering. 

That is why I set it as false.